catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlUtil.java

```diff
@@ -0,0 +1,174 @@
+      (oldMetacard, newMetacard) ->
+          !Sets.symmetricDifference(
+                  attributeToSet.apply(oldMetacard, Security.ACCESS_ADMINISTRATORS),
+                  attributeToSet.apply(newMetacard, Security.ACCESS_ADMINISTRATORS))
+              .isEmpty();
+
+  /**
+   * Does a diff between the old set of {@link Metacard} access-individuals with the set to see if
+   * anyone was added or removed.
+   */
+  public static BiFunction<Metacard, Metacard, Boolean> accessIndividualsHasChanged =
+      (oldMetacard, newMetacard) ->
+          !Sets.symmetricDifference(
+                  attributeToSet.apply(oldMetacard, Security.ACCESS_INDIVIDUALS),
+                  attributeToSet.apply(newMetacard, Security.ACCESS_INDIVIDUALS))
+              .isEmpty();
+
+  /**
+   * Does a diff between the old set of {@link Metacard} access-groups with the set to see if a role
+   * was added or removed.
+   */
+  public static BiFunction<Metacard, Metacard, Boolean> accessGroupsHasChanged =
+      (oldMetacard, newMetacard) ->
+          !Sets.symmetricDifference(
+                  attributeToSet.apply(oldMetacard, Security.ACCESS_GROUPS),
+                  attributeToSet.apply(newMetacard, Security.ACCESS_GROUPS))
+              .isEmpty();
+
+  /**
+   * Pulls off a list of values associated with a particular attribute on a {@link Metacard}
+   *
+   * @param metacard that you want to pull values from
+   * @param attributeName that you are targeting to pull off in the input {@link Metacard}
+   * @return Populated list of string values if attribute was found, otherwise empty list
+   */
+  public static List<String> getValuesOrEmpty(Metacard metacard, String attributeName) {
+    Attribute attribute = metacard.getAttribute(attributeName);
+    if (attribute != null) {
+      return attribute.getValues().stream().map(String::valueOf).collect(Collectors.toList());
+    }
+    return Collections.emptyList();
+  }
+
+  /**
+   * @return List of access-individuals from input {@link Metacard} if they exist, otherwise return
+   *     an empty set
+   */
+  public static Set<String> getAccessIndividuals(Metacard metacard) {
+    return attributeToSet.apply(metacard, Security.ACCESS_INDIVIDUALS);
+  }
+
+  /**
+   * @return List of access-groups from input {@link Metacard} if they exist, otherwise return an
+   *     empty set
+   */
+  public static Set<String> getAccessGroups(Metacard metacard) {
+    return attributeToSet.apply(metacard, Security.ACCESS_GROUPS);
+  }
+
+  /**
+   * @return List of access-administrators from input {@link Metacard} if they exist, otherwise
+   *     return an empty set
+   */
+  public static Set<String> getAccessAdministrators(Metacard metacard) {
+    return attributeToSet.apply(metacard, Security.ACCESS_ADMINISTRATORS);
+  }
+
+  /** Sets owner value associated with a particular {@link Metacard} */
+  public static Metacard setOwner(Metacard metacard, String email) {
```

Comment:
✏️ Recommend renaming the String, it is no longer required to be an email address

Reply:
In most cases, this will be an email address. I can rename to subjectIdentity